nufsi.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
```

Comment:
just curious, what's this script for?

Reply:
I included here by mistake. It is from [this blog](https://two-wrongs.com/referencing-nuget-packages-in-fsharpi)
I will remove from this PR later.